config/roles.php

```diff
@@ -75,6 +75,7 @@
         'rfidTags.edit.all',
         'rfidTags.destroy',
         'pins.view.all',
+        'pins.view.own',
```

Comment:
sorry small nitpick that should have been `pins.view.self` to match the existing convention

Reply:
dont, thanks!

Reply:
that commit added an extra file that should not have been committed